### PR TITLE
tests for input test

### PR DESCRIPTION
**Pull request by JanetJanx**
Need some help on how to test the number function with input

### CODE REVIEW

**Comment by JanetJanx** [comment]
Hello @dennisja , could you please look at this also

File: user_category.py [hunk 1]
```diff
@@ -28,26 +28,24 @@ def number():
...
     if user_category_value < 18:
-        print ("Your user category is: Minor")
-        return "Your user category is: Minor"
+        cat_message = "Your user category is: Minor"
+        return cat_message
 
     elif user_category_value <= 36:
-        print ("Your user category is: Youth")
-        return "Your user category is: Youth"
+        cat_message = "Your user category is: Youth"
```
Comment (dennisja) [inline user_category.py:42]: Why did you choose to create a variable?
I personally find it better to create a variable only if I'm going to use it in many places

File: user_tests.py [hunk 2]
```diff
@@ -1,15 +1,12 @@
-import user_category
+from user_category import user_type
```
Comment (dennisja) [inline user_tests.py:1]: Why not import both number and user_type from user_category forexample
```python
from user_category import user_type, number
```

**Comment by dennisja** [comment]
Your tests are still few and their coverage is still low
Remind me tomorrow to show you how you can see your test coverage and what it means

**Comment by dennisja** [comment]
You should by the way remove the `user_category.pyc` file from your repo

File: user_category.py (hunk 1 above)
Comment (JanetJanx) [reply user_category.py:42]: Alright thanks, I have removed it

File: user_tests.py (hunk 2 above)
Comment (JanetJanx) [reply user_tests.py:1]: @dennisja , I think your help today through TDD will be of great value coz I have questions on testing number function with input

Comment (dennisja) [reply user_tests.py:1]: Hope so, will be waiting to see and talk to you then

Comment (JanetJanx) [reply user_tests.py:1]: Alright thanks

**Pull request merged**